src/components/stepindicator/StepIndicator/StepIndicator.stories.tsx

```diff
@@ -64,3 +64,13 @@ export const smallCounters = (): React.ReactElement => (
     <StepIndicatorStep label="Review and submit" />
   </StepIndicator>
 )
+
+export const heading = (): React.ReactElement => (
```

Comment:
```suggestion
export const customHeadingLevel = (): React.ReactElement => (
```

Naming the story after what its demonstrating.